src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -254,6 +254,16 @@ public static bool IsWindowsAzureManagedHosting(this IEnvironment environment)
             return environment.IsAppService() && RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the application is running in Centauri environment.
+        /// </summary>
+        /// <param name="environment">The environment to verify.</param>
+        /// <returns><see cref="true"/> if running in Managed App environment; otherwise, false.</returns>
+        public static bool IsManagedAppEnvironment(this IEnvironment environment)
```

Comment:
Add unit tests to EnvironmentExtensionsTests for any new logic you're introducing